python/simple/v1/ampli/ampli.py

```diff
@@ -0,0 +1,818 @@
+# 
+# ampli.py
+# 
+# Ampli - A strong typed wrapper for your Analytics
+# 
+# This file is generated by Amplitude.
+# To update run 'ampli pull python-ampli'
+# 
+# Required dependencies: amplitude-analytics
+# Tracking Plan Version: 0
+# Build: 0.0.0
+# Runtime: python-ampli
+# 
+# [View Tracking Plan](https://data.amplitude.com/test-codegen/Test%20Codegen/events/main/latest)
+# 
+# [Full Setup Instructions](https://data.amplitude.com/test-codegen/Test%20Codegen/implementation/main/latest/getting-started/python-ampli)
+# 
+# pylint: skip-file
+# flake8: noqa
+
+
+import logging
+import enum
+from typing import Dict, Optional, List, Any, Union
+
+from amplitude import *
+
+
+class Environment(enum.Enum):
+    PRODUCTION = 'production'
+    DEVELOPMENT = 'development'
+    
+
+class LoadClientOptions:
+    
+    def __init__(self, api_key: Optional[str] = None,
+                 instance: Optional[Amplitude] = None,
+                 configuration: Optional[Config] = None):
+        self.api_key = api_key
+        self.instance = instance
+        self.configuration = configuration
+
+
+class LoadOptions:
+
+    def __init__(self, environment: Environment = None,
+                 disabled: bool = False,
+                 client: LoadClientOptions = None):
+        self.environment = environment
+        self.disabled = disabled
+        self.client = client
+        
+        
+class IdentifyProperties:
+    """Identify
+
+    [View in Tracking Plan](https://data.amplitude.com/test-codegen/Test%20Codegen/events/main/latest/Identify)
+
+    Identify properties.
+
+    :param required_number: Description for identify requiredNumber
+    :param optional_array: Description for identify optionalArray
+    """
+
+    def __init__(
+        self,
+        required_number: float,
+        optional_array: Optional[List[str]] = None
+    ):
+        self.properties = {
+            "optionalArray": optional_array,
+            "requiredNumber": required_number
+        }
+
+
+class GroupProperties:
+    """Group
+
+    [View in Tracking Plan](https://data.amplitude.com/test-codegen/Test%20Codegen/events/main/latest/Group)
+
+    Group properties.
+
+    :param required_boolean: Description for group requiredBoolean
+    :param optional_string: Description for group optionalString
+    """
+
+    def __init__(
+        self,
+        required_boolean: bool,
+        optional_string: Optional[str] = None
+    ):
+        self.properties = {
+            "optionalString": optional_string,
+            "requiredBoolean": required_boolean
+        }
+
+
+API_KEY: Dict[Environment, str] = {
+    Environment("development"): "",
+    Environment("production"): ""
+}
+DEFAULT_CONFIGURATION = Config(
+    server_zone='US',
+    plan=Plan(
+        branch="main",
+        source="python-ampli",
+        version="0",
+        version_id="79154a50-f057-4db5-9755-775e4e9f05e6"
+    )
+)
```

Comment:
Can you please move these to directly under the Environment class definition. I think it is helpful to have all the options and keys at the top of the file before the Events/Properties

https://github.com/amplitude/ampli-examples/blob/main/node/typescript/src/ampli/index.ts#L25-L42